annotations.md

```diff
@@ -40,6 +40,15 @@ This specification defines the following annotation keys, intended for but not l
     ```
 * **org.opencontainers.image.title** Human-readable title of the image (string)
 * **org.opencontainers.image.description** Human-readable description of the software packaged in the image (string)
+* **org.opencontainers.image.base.digest** [Digest](descriptor.md#digests) of the image this image is based on (string)
+  * This SHOULD be the immediate image sharing zero-indexed layers with the image, such as from a Dockerfile `FROM` statement.
+  * This SHOULD NOT reference any other images used to generate the contents of the image (e.g., multi-stage Dockerfile builds).
+* **org.opencontainers.image.base.name** Image reference of the image this image is based on (string)
+  * This SHOULD be image references in the format defined by [distribution/distribution](https://github.com/distribution/distribution/blob/d0deff9cd6c2b8c82c6f3d1c713af51df099d07b/reference/reference.go).
+  * This SHOULD be a fully qualified reference name, without any assumed default registry. (e.g., `index.docker.io/my-org/my-image:tag` instead of `my-org/my-image:tag`).
```

Comment:
"index." might not be needed

Reply:
This was added based on feedback from @SteveLasker -- Steve, it doesn't seem like the assumed default registry is defined in the spec anywhere. Should this be `docker.io` or `index.docker.io`?

Reply:
docker uses a few urls for pulling content. `docker.io/my-org/my-image:tag` would suffice.
For instance, these all work:
- `docker pull index.docker.io/wabbitnetworks/net-monitor:v1`
- `docker pull docker.io/wabbitnetworks/net-monitor:v1`
- `docker pull wabbitnetworks/net-monitor:v1`

The spec doesn't reference the default registry, which is actually a specific docker client implementation I'm trying to help folks move away from. (random plug to help drive registry configuration forward: (https://github.com/SteveLasker/drafts/blob/main/registry-repo-config.md)

Reply:
I understand the concept of a default registry, I was specifically asking whether the spec does now, or should in the future, specify what the default registry is. `docker.io` and `index.docker.io` both work, but which one should OCI specify is the value?

We don't have to define "default registry" in this PR, but we can hint at it with the example value. Should it be `docker.io` or `index.docker.io`?

Or, we can remove the "e.g.," parenthetical, and avoid having to answer the question at all.

Reply:
I'm trying to make this a bit more simple, and avoid any assumptions about the URL with a concrete example. 
The example could say `registry.example.com/org/artifact:tag`, but folks that reference content from docker hub might think they can put in `ubuntu` and be done. Should anyone reading this assume `ubuntu` came from docker hub? What I was hoping to accomplish was just use a docker hub example, but show how users should always use the fully qualified registry URL they actually based the reference upon. This way there are no assumptions made. And, any consumers of the annotation would know exactly where it was derived from. They may choose to get the image from somewhere else, but they at least knew where the reference was made, including the digest used.

Reply:
I would avoid using Docker Hub as the example, if we want one. Something like `registry.example.com` would be preferable, really.

Reply:
Good call, done.

Reply:
Not relevant anymore, but for future reference, `index.docker.io` is a legacy domain and the preference appears to be `docker.io` according to distribution/distribution (and now I have something else to fix in my side project): https://github.com/distribution/distribution/blob/90e62ac24c16a8518e20390ce8d9aa4fbf6ee15f/reference/normalize.go#L13